### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by shegy28**
### Desktop version of the Portfolio
**In this branch, I added:**

- Added desktop image to ./images folder
- Styled the portfolio to fit a desktop using @media query
- Gave it a breakpoint using (min-width).
- Breakpoint starts from 768px.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -447,3 +446,280 @@ input::placeholder {
...
+
+  .nav {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+  }
```
Comment (emmiiorji) [inline styles.css:505]: Your nav bar looks great. As represented by the design, kindly make the nav bar to be fixed while the user scrolls. A look at the representation below in the design shows that's what's expected.

Figma design:
image

File: styles.css [hunk 2]
```diff
@@ -447,3 +446,280 @@ input::placeholder {
...
+  .main {
+    display: block;
+    background-image: url(./images/Header\ bg@2x.png);
+    height: 90vh;
+    width: 100%;
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 1px;
+  }
```
Comment (emmiiorji) [inline styles.css]: > [DEPRACATED] You have well implemented the hero section, however, there're a few things to point out for little adjustment in order to match the design template.
> 
> - The whole content of the hero section should be visible on height of 700px
> - On the design, the text content has a little more width


Your design:
image


Figma design:
image

File: index.html [hunk 3]
```diff
@@ -157,6 +260,42 @@ <h4 class="info-icons-p2">2015</h4>
...
+                            </div>
+                            <div class="pointer-div">
+                                <div class="button-div">
+                                    <button class="see-project1">
+                                        <a href="#">See Project</a>
+                                    </button>
+                                </div>
+                                <div class="hand-pointer">
+                                    <img src="./images/hand-cursor 1.png" alt="">
+                                </div>
```
Comment (emmiiorji) [inline index.html]: Your effort toward implementing this design is well recognized. However, the image below in the design only represents how the work cards should look when hovered over but without the mouse image. Good a thing, you've implemented it already, you only have to use it on hover.

image

File: styles.css [hunk 4]
```diff
@@ -447,3 +446,280 @@ input::placeholder {
...
+
+  .see-project1 {
+    padding: 12px 8px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    margin: 20px 0;
+    background: #4053fc;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  }
```
Comment (emmiiorji) [inline styles.css]: Kindly implement the button interactions as given by the design.

Figma button interactions:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @shegy28, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Highlights✔

- Meaningful commit messages
- Linter checks are passing 
- Adhered to GitHub flow
- PR is descriptive
- Great design


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -20,6 +20,17 @@ <h3><a href="#">Asouzu Chimezie</a></h3>
                 <div class="menu">
                     <img src="/images/Union.png" alt="hamburger icon">
                 </div>
+                <ul class="desktop-header">
```
Comment (emmiiorji) [inline index.html]: It seems like you have referred to an image that is not found in the images directory as the hamburger. Kindly confirm this.

image

File: styles.css (hunk 2 above)
Comment (shegy28) [reply styles.css]: @emmiiorji Thanks for the feedback but if you pay close attention to the width on the screen shot of my website as compared to that on figma you will notice that mine is 1075px and that of figma is 1440px.  Try again please but this time with a with the same width and see the difference. thank you

Comment (emmiiorji) [reply styles.css]: Hi @shegy28,

Rightly said, thanks for pointing that. Kindly ignore this comment and work on other comments, I'll mark it as deprecated.

Comment (shegy28) [reply styles.css]: Hi @emmiiorji thanks

Comment (emmiiorji) [reply styles.css]: Hi @shegy28,

On another look, I noticed that it seems not properly responsive.

This seems to stem from the margin given to the wrapper class as shown:
image


I recommend that you use a percentage value for the right and left margins e.g `margin: 200px 18%`.

Comment (shegy28) [reply styles.css]: @emmiiorji Now i am honestly confused and dont understand what you want me to do

Comment (shegy28) [reply styles.css]: @emmiiorji  I have implemented the required changes to fit that of the aspect ratio of the template on figma which is 1440 X 916. Please i would appreciate if i am been reviewed that way thanks.

Comment (emmiiorji) [reply styles.css]: @shegy28 I already marked this thread as deprecated, so you may not worry much.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @shegy28 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**